java/examples/BUILD

```diff
@@ -0,0 +1,23 @@
+package(default_visibility = ["//visibility:public"])
+
+load("@io_bazel_rules_docker//docker:docker.bzl", "docker_build")
+
+docker_build(
+    name = "image",
+    base = "//java:jetty",
+    directory = "/jetty/webapps",
+    files = [":helloworld.war"],
+    workdir = "/jetty",
```

Comment:
I'd move this into the base.  We could add the jetty-demo as an example here with:
```
docker_build(
   name = "jetty-demo",
   base = "//java:jetty",
   workdir = "/jetty/jetty-demo",
)
```